docs/DeveloperGuide.md

```diff
@@ -764,13 +764,20 @@ testers are expected to do more exploratory testing.
 
 ### 7.4 Editing an applicant
 
-1. tc description
+1. Editing one parameter of an applicant on InternConnect
 
-   1. Prerequisites:
+   1. Prerequisites: One applicant without the email `johndoe@example.com`
```

Comment:
You might want to use another example of edit, maybe edit university?
Cause here it might cause a duplicate person

Reply:
changed to university.